code/game/machinery/constructable_frame.dm

```diff
@@ -26,6 +26,28 @@
 			D = "Requires [english_list(component_list)]."
 		desc = D
 
+/obj/machinery/constructable_frame/verb/rotate()
+	set name = "Rotate Frame Counter-Clockwise"
+	set category = "Object"
+	set src in oview(1)
+
+	if(usr.incapacitated())

```

Comment:
This should probably also have `can_use()` or whatever it is. In order to check if mob can use, so mobs like ghosts or mice or dead people won't rotate it

Reply:
```suggestion
	if(use_check(usr) || anchored)
```